src/urbit/mar/chess/pals.hoon

```diff
@@ -0,0 +1,18 @@
+=,  format
+|_  pals=(set @p)
+++  grab
+  |%
+  ++  noun  pals
+  --
+++  grow
+  |%
+  ++  noun  pals
+  ++  json
+    %-  frond:enjs
+    :-  'mutuals'
+    :-  %a
+    %-  turn  pals
+    |=(fren=@p (ship:enjs fren))
```

Comment:
Just realised after pushing this change I'm trying to `turn` a set, which will fail. Will fix.